api/v1/server/handlers/v1/cel/debug.go

```diff
@@ -25,6 +31,19 @@ func (c *V1CELService) V1CelDebug(ctx echo.Context, request gen.V1CelDebugReques
 	),
 	)
 
+	if err != nil {
```

Comment:
Isn't the point of the debug endpoint to return a readable error to the user here? Why would we ingest this into the table?

Reply:
🤦 yeah I guess no need, will remove this one